src/arch/riscv/pmp/pmp.c

```diff
@@ -17,7 +17,7 @@
  * @param reg The index of the PMP configuration register to write to.
  * @param value The value to write to the register.
  */
-static inline void write_pmpcfg(uint32_t reg, uint32_t value) {
+void write_pmpcfg(uint32_t reg, uint32_t value) {
```

Comment:
Do you want to make these routines as global (global view)

Reply:
Yes, I do want to make these routines global. I’ll update the declarations to use extern for the global view and ensure the definitions are appropriately handled. I’ll commit these changes shortly.